docs/source/api/python/mantid/geometry/DetectorInfo.rst

```diff
@@ -0,0 +1,15 @@
+==============
+ DetectorInfo
+==============
+
+This is a python binding to the C++ class Mantid::Geometry::DetectorInfo.
```

Comment:
What is it for? Why would you need it? How does it link with other types?

Reply:
I'll add in a description of what the DetectorInfo class is used for.